src/fedora/28/helix/amd64/Dockerfile

```
@@ -26,7 +26,7 @@ RUN python -m pip install --upgrade pip==19.0.3 && \
 ENV LANG=en-US.UTF-8
 
 # create helixbot user and give rights to sudo without password
-RUN /usr/sbin/useradd -G adm -s /bin/sh helixbot; \
+RUN /usr/sbin/adduser --disabled-password --gecos '' --uid 1001 --shell /bin/bash --ingroup adm helixbot
```

FYI - the Fedora CI legs are failing.  
```suggestion
RUN /usr/sbin/adduser --disabled-password --gecos '' --uid 1001 --shell /bin/bash --ingroup adm helixbot; \
```

---

fixed. The docker file explicitly refers to mcr but it is failing to find dependent image. 

```
docker build .
Step 1/6 : FROM mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-30
manifest for mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-30 not found
```

---

How are you building?  You need to have the prereqs built (e.g. mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-30)  - the build script handles this for you.
  
```
\build.ps1 -DockerfilePath "fedora/30*"
```